pvlib/irradiance.py

```diff
@@ -745,6 +744,10 @@ def haydavies(surface_tilt, surface_azimuth, dhi, dni, dni_extra,
         defined as degrees from horizontal (e.g. surface facing up = 0,
         surface facing horizon = 90)
 
+    surface_tilt : numeric
+        Panel tilt from the horizontal, in decimal degrees, see
+        :term:`surface_tilt`.
```

Comment:
add `see :term:`surface_tilt` with the full definition, or shorten definition (like this example) and add it?

Reply:
I'd rather have short definitions to get rid of small inconsistencies in the redactions. And that it's kind of simple it's definition, no need to clutter it.

Reply:
We are in agreement with that; if I understood correctly, Dax was referring to the full definition:
```
surface_tilt : numeric
        Surface tilt angles in decimal degrees. The tilt angle is
        defined as degrees from horizontal (e.g. surface facing up = 0,
        surface facing horizon = 90)
```
It does not appear on the diff by default, just gotta click on those expand arrows.